conf/config_data_analysis.yaml

```diff
@@ -0,0 +1,107 @@
+# Deep learning configuration file ------------------------------------------------
```

Comment:
can config_data_analysis and config.yaml be merged together. Seems like alot of duplicates. Can we just add the data_analysis part to config.yaml?

Reply:
Absolutely! I will make the changes to the config.yaml.

Reply:
Great! Thanks

Reply:
Should be all good now.